plugins/single_plugins/scale.cpp

```diff
@@ -142,10 +142,10 @@ class wayfire_scale : public wf::plugin_interface_t
             return;
         }
 
-        scale_data[view].transformer = new wf_scale(view);
-        view->add_transformer(std::unique_ptr<wf_scale>(
-            scale_data[view].transformer), transformer_name);
-        scale_data[view].transformer->alpha = 1;
+        wf_scale *tr = new wf_scale(view);
+        scale_data[view].transformer = tr;
+        view->add_transformer(std::unique_ptr<wf_scale>(tr), transformer_name);
+        tr->alpha = 1;
```

Comment:
I know it was like this already but I think the transformer values are already initialized, so no need to set alpha = 1 here.

Reply:
Yes, I see and agree; I'll remove the assignment